app/models/result_contact.rb

```diff
@@ -4,17 +4,17 @@ def self.top_level_label
     '//point_of_contact'
```

Comment:
search if method is being used somewhere else and if not, remove it from this class

Reply:
Removed top_level_label method.

Reply:
RSpec Test stil PASS:

<img width="778" alt="Screen Shot 2024-02-12 at 8 48 04 PM" src="https://github.com/ctti-clinicaltrials/aact/assets/81119399/b6c9c492-e4fa-4744-8e9c-9367f746c06e">